config/webhook/manifests.yaml

```diff
@@ -34,6 +34,7 @@ webhooks:
     failurePolicy: Fail
     name: inferenceservice.kserve-webhook-server.pod-mutator
     sideEffects: None
+    reinvocationPolicy: IfNeeded
```

Comment:
Can we also update the helm chart configuration here
https://github.com/kserve/kserve/blob/master/charts/kserve-resources/templates/webhookconfiguration.yaml#L36

Reply:
Yes. I fixed.